contracts/message-bus/MessageOutbox.sol

```diff
@@ -89,6 +89,7 @@ contract MessageOutbox is MessageBox {
     )
         internal
     {
+        // It is used to ensure that ConsensusCoGateway can only be setup once.
```

Comment:
Remove this comment. 
`MessageOutbox` is very generic and not specific to `ConsensusCoGateway `

Reply:
Done